### ISSUE TITLE
cli: debug.zip: want ability to trigger debug.zip collection from console

### ISSUE THREAD

**Issue by andreimatei** [cockroachdb/cockroach#67516]
More than once I've seen users fumble to produce debug.zips under pressure. Certificates are not found, the exact command line is not clear. Even under the best of circumstances ssh'ing through hop after hop is time consuming. We need to make it easier to collect this file. I'd like to be able to trigger the collection from the Console, and have the file written on some node's disk. Bonus points if we also automatically upload the debug.zip to SendSafely or wherever (firewall permitting); uploading the damn thing is another thing that usually takes forever.

cc @knz for triage :P

**Comment by thtruo** [comment]
That's correct (for reference see [CC-5064](https://cockroachlabs.atlassian.net/browse/CC-5064))

**Comment by knz** [comment]
Thanks andrei, that's the input we needed.
